core/commands/id.go

```diff
@@ -44,6 +37,10 @@ type IdOutput struct {
 const (
 	formatOptionName   = "format"
 	idFormatOptionName = "peerid-base"
+	// FIXME(BLOCKING): The "offline" name might be colliding with the daemon
+	//  option of the same name or a similar option, getting:
+	//  Error: option name "offline" used multiple times
+	offlineOptionName = "id-offline"
```

Comment:
The option already exists globally in 

https://github.com/ipfs/go-ipfs/blob/91c52657166bcf86f2476926e4fe56694dc26562/core/commands/root.go#L25

I'll reference that and fix this.

Reply:
:+1:  on removing `id-offline` and leveraging `offline` that is registered at the root.